app/views/providers/bank_statements/_uploaded_files.html.erb

```diff
@@ -0,0 +1,29 @@
+<table class="govuk-table <%= 'hidden' if attachments.empty? %>">
+  <caption class="govuk-table__caption govuk-visually-hidden"><%= t('.title') %></caption>
+  <thead class="govuk-table__head">
+  <tr class="govuk-table__row">
+    <th class="govuk-table__header" scope="col"><%= t('.filename') %></th>
+    <th class="govuk-table__header" scope="col"><%= t('.status') %></th>
+    <td class="govuk-table__header" scope="col"></td>
+  </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+
+  <% attachments.order(:created_at).each do |attachment| %>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell"><%= attachment.document.filename %></td>
+      <td class="govuk-table__cell"><span class="govuk-tag"><%= t('.uploaded') %></span></td>
```

Comment:
the prototype doesn't have this tag field. It shows size instead, although I am not opposed to including it as well as the size. replacing line 15 with the following should work:
 <td class="govuk-table__cell"><%= number_to_human_size(attachment.document.byte_size) %></td>

Reply:
to be queried with design as statement of case and uploaded evidence pages should be consistent with this